src/griffe/docstrings/numpy.py

```diff
@@ -71,6 +71,7 @@
     "receives": DocstringSectionKind.receives,
     "raises": DocstringSectionKind.raises,
     "warns": DocstringSectionKind.warns,
+    "example": DocstringSectionKind.examples,
```

Comment:
There's no "admonition" feature with the Numpy parser (IIRC), but for the sake of consistency I believe we should drop this as well (also, maybe it'll support admonitions later).